source/features/list-prs-for-file.tsx

```diff
@@ -23,35 +23,54 @@ function getHovercardUrl(prNumber: number): string {
 	return buildRepoURL('pull', prNumber, 'hovercard');
 }
 
+const buttonId = 'rgh-list-prs-for-file-button';
+const popoverId = 'rgh-list-prs-for-file-popover';
+
 function getDropdown(prs: number[]): HTMLElement {
 	const isEditing = pageDetect.isEditingFile();
 	const icon = isEditing
-		? <AlertIcon className="v-align-middle color-fg-attention"/>
-		: <GitPullRequestIcon className="v-align-middle"/>;
-	// Markup copied from https://primer.style/css/components/dropdown
+		? <AlertIcon className="color-fg-attention"/>
+		: <GitPullRequestIcon/>;
+
 	return (
-		<details className="dropdown details-reset details-overlay flex-self-center rgh-list-prs-for-file">
-			<summary className="btn btn-sm">
+		<div>
+			<button
+				type="button"
+				className="Button Button--secondary color-fg-muted"
+				id={buttonId}
+				// @ts-expect-error Used by GitHub
```

Comment:
I tried to add `popovertarget ` to `IntrinsicAttributes` with no luck, TS still complains 🤷

Reply:
I forgot this is HTML standard now (thought it was a GitHub thing), I really only read it one time or two.